<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillGroupManagement</class>
 <widget class="QDialog" name="SkillGroupManagement">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblSkillGrp">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Skill Group List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblSubSkill">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Sub Skills</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="SkillGrpListWgt">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>75</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="SubSkillListWgt">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>75</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pbtnDelSubSkill">
     <property name="text">
      <string>Delete Sub Skill</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pbtnDelSkillGrp">
     <property name="text">
      <string>Delete Skill Group</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pbtnAddSkillGrp">
     <property name="text">
      <string>Add Skill Group</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbtnAddSubSkill">
     <property name="text">
      <string>Add Sub Skill</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
